DetectorDescription/DDCMS/test/BuildFile.xml

```diff
@@ -4,6 +4,16 @@
   <use name="dd4hep"/>
 </bin>
 
+<bin name="testDD4hepFilteredViewLevel" file="DDFilteredView.level.cppunit.cc,testRunner.cpp">
```

Comment:
@ianna I'm a bit confused. Are these `cppunit` files included in other PR?

Reply:
I see them here: https://github.com/cms-sw/cmssw/blob/e8a34be27ad271d0e4c56883e9f190d228a99b67/DetectorDescription/DDCMS/test/DDFilteredView.level.cppunit.cc

Reply:
Not a problem, I’ll rebase the second in.